vms/avm/vm_test.go

```diff
@@ -35,10 +35,7 @@ func TestInvalidGenesis(t *testing.T) {
 	vm := &VM{}
 	ctx := snowtest.Context(t, snowtest.XChainID)
 	ctx.Lock.Lock()
-	defer func() {
-		require.NoError(vm.Shutdown(context.Background()))
```

Comment:
We should probably inspect this for consistency (and clearly document the requirements)... But the chain manager doesn't call `Shutdown` on a VM if `Initialize` fails.